[D] 2023-10-28 17:19:52.652 [FileLogger.kt:86]: FileLogger reset
[D] 2023-10-28 17:23:19.326 [MainActivity.kt:120]: Logging something...
[D] 2023-10-28 17:25:12.861 [LogTests.kt:15]: TEST-LOG - 1a - MainActivity onCreate was just called
[E] 2023-10-28 17:25:12.863 [LogTests.kt:16]: TEST-LOG - 1b - MainActivity onCreate was just called
java.lang.Exception: TEST
	at com.michaelflisar.lumberjack.demo.LogTests.run(LogTests.kt:16)
	at com.michaelflisar.lumberjack.demo.ui.MainActivity$Content$1.invokeSuspend(MainActivity.kt:43)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performTrampolineDispatch(AndroidUiDispatcher.android.kt:81)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(AndroidUiDispatcher.android.kt:57)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

[E] 2023-10-28 17:25:12.868 [LogTests.kt:24]: TEST-LOG - 3 - Some error message
[E] 2023-10-28 17:25:12.869 [LogTests.kt:25]: 
TEST-LOG - 4 - TestException (some info)
	at com.michaelflisar.lumberjack.demo.LogTests.run(LogTests.kt:25)
	at com.michaelflisar.lumberjack.demo.ui.MainActivity$Content$1.invokeSuspend(MainActivity.kt:43)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performTrampolineDispatch(AndroidUiDispatcher.android.kt:81)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(AndroidUiDispatcher.android.kt:57)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

[D] 2023-10-28 17:25:12.872 [LogTests.kt:32]: TEST-LOG - 5 - from within lambda: func call 1...
[D] 2023-10-28 17:25:12.867 [LogTests.kt:17]: TEST-LOG - 1c - MainActivity onCreate was just called
[D] 2023-10-28 17:25:12.873 [LogTests.kt:32]: TEST-LOG - 5 - from within lambda: func call 2...
[D] 2023-10-28 17:25:12.875 [LogTests.kt:32]: TEST-LOG - 5 - from within lambda: func call 3...
[D] 2023-10-28 17:25:12.876 [LogTests.kt:44]: TEST-LOG - Test 0
[D] 2023-10-28 17:25:12.877 [LogTests.kt:40]: TEST-LOG - 6 - from within coroutine on background thread: Thread[DefaultDispatcher-worker-10,5,main]
[D] 2023-10-28 17:25:12.878 [LogTests.kt:44]: TEST-LOG - Test 1
[D] 2023-10-28 17:25:12.879 [LogTests.kt:44]: TEST-LOG - Test 2
[D] 2023-10-28 17:25:12.881 [LogTests.kt:44]: TEST-LOG - Test 3
[D] 2023-10-28 17:25:12.882 [LogTests.kt:44]: TEST-LOG - Test 4
[D] 2023-10-28 17:25:12.883 [LogTests.kt:44]: TEST-LOG - Test 5
[D] 2023-10-28 17:25:12.885 [LogTests.kt:44]: TEST-LOG - Test 6
[D] 2023-10-28 17:25:12.886 [LogTests.kt:44]: TEST-LOG - Test 7
[D] 2023-10-28 17:25:12.887 [LogTests.kt:44]: TEST-LOG - Test 8
[D] 2023-10-28 17:25:12.888 [LogTests.kt:44]: TEST-LOG - Test 9
[D] 2023-10-28 17:25:12.889 [LogTests.kt:44]: TEST-LOG - Test 10
[D] 2023-10-28 17:25:12.890 [LogTests.kt:58]: TEST-LOG - Debug Message from library...
[D] 2023-10-28 17:25:12.892 [LogTests.kt:61]: TEST-LOG - Debug log...
[E] 2023-10-28 17:25:12.896 [LogTests.kt:62]: TEST-LOG - Error log...
java.lang.Exception: EX
	at com.michaelflisar.lumberjack.demo.LogTests.run(LogTests.kt:62)
	at com.michaelflisar.lumberjack.demo.ui.MainActivity$Content$1.invokeSuspend(MainActivity.kt:43)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performTrampolineDispatch(AndroidUiDispatcher.android.kt:81)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performTrampolineDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.run(AndroidUiDispatcher.android.kt:57)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)

[V] 2023-10-28 17:25:12.920 [LogTests.kt:48]: TEST-LOG - Verbose log...
[D] 2023-10-28 17:25:12.926 [LogTests.kt:49]: TEST-LOG - Debug log...
[I] 2023-10-28 17:25:12.931 [LogTests.kt:50]: TEST-LOG - Info log...
[W] 2023-10-28 17:25:12.937 [LogTests.kt:51]: TEST-LOG - Warn log...
[E] 2023-10-28 17:25:12.942 [LogTests.kt:52]: TEST-LOG - Error log...
[WTF] 2023-10-28 17:25:12.949 [LogTests.kt:53]: TEST-LOG - WTF log...